en/oop.md

```diff
@@ -43,6 +43,42 @@ The following class modifiers are supported:
     
     }
 
+<a name='classes-interfaces'></a>
+### Implementing Interfaces
+Zephir classes can implement any number of interfaces, provided that these interfaces are visible for the extension to use. However there are times that your extension might require to implement an interface that is built in another extension or package. 
+
+If we want to implement the `MiddlewareInterface` from the `PSR` package, we will need to:
+
+	// middlewareinterfaceex.zep
+	namespace Test\Oo\Extend;
+ 	
+ 	use Psr\Http\Server\MiddlewareInterface;
+ 	
+ 	interface MiddlewareInterfaceEx extends MiddlewareInterface
+	{
+	
+	}
+
+From here we can use the "stub" interface throughout our extension. 
+
+	/**
+     * @test
+     * @issue https://github.com/phalcon/zephir/issues/1686
```

Comment:
Do we want to include this line in the docs?

Reply:
Removed the @issue